recommendation/templates/recommendation/recommendation_form.html

```diff
@@ -7,41 +7,92 @@
 <div style="margin-left:20%;height: 90%; margin-bottom: 5%; width:50%; margin-top: 2%;">
     <h2>Teacher/Mentor Recommendations</h2>
     <h6>Please enter the name and email address of the person that you are requesting to fill out your recommendation. </h6>
```

Comment:
For better UX, since there's no form displayed after 2 recommendation requests we can hide this and the next message.